get-artifacts/README.md

```diff
@@ -0,0 +1,60 @@
+# Get Artifacts #
+
+A downloader for artifacts from multiple prior workflow_runs (by workflow_file_name and branch), or a single run by workflow_run_id.
```

Comment:
`workflow_file name` I think?

Reply:
Indeed, I tried adding documentation about it :D